releasenotes/notes/50328.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
```

Comment:
```suggestion
kind: bugfix
```

IMO this is just something that didn't work correctly before, and should have, and thus this is a bugfix - do you have any objection to classifying it as such?

Reply:
No objections, changed.